doc/getting-started/getting-started.rst

```diff
@@ -438,7 +426,7 @@ To generate a scenario configuration file and launch script:
          :class: drop-shadow
 
    #. In the dialog box, select **shared** as the default scenario setting and
-      then click **OK**.
+      then click **OK**. This sample default scenario defines six user VMs.
```

Comment:
```suggestion
      then click **OK**. This sample default scenario defines six User VMs.
```
We tend to use a capital U when writing `User VM`, any reason we shouldn't here?